docs/plugins/spotify.rst

```diff
@@ -67,10 +106,20 @@ in config.yaml under the ``spotify:`` section:
   track/album/artist fields before sending them to Spotify.  Can be useful for
   changing certain abbreviations, like ft. -> feat.  See the examples below.
   Default: None.
+- **tokenfile**: Filename of the JSON file stored in the beets configuration
+  directory to use for caching the OAuth access token.
+  Default: ``spotify_token.json``.
```

Comment:
Hmm… perhaps we shouldn't even mention this option in the docs. I don't expect anyone will ever need to change it, and omitting the unnecessary options should make the docs page a little easier to read. (FWIW, we don't document `tokenfile` for the other OAuth-based plugins we have.)